Needs to be changed to not pass the obfuscation flag:
```
chainparams.cpp:160:104: error: no matching function for call to ‘CDNSSeedData::CDNSSeedData(const char [25], const char [30], bool)’
chainparams.cpp:160:104: note: candidates are:
chainparams.h:19:5: note: CDNSSeedData::CDNSSeedData(const string&, const string&)
chainparams.h:19:5: note: candidate expects 2 arguments, 3 provided
chainparams.h:17:8: note: CDNSSeedData::CDNSSeedData(const CDNSSeedData&)
chainparams.h:17:8: note: candidate expects 1 argument, 3 provided
```

--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -157,6 +157,7 @@ public:
         vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org")); // Luke Dashjr
         vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com")); // Addy Yeow
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org")); // Jeff Garzik
+        vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch")); // Jonas Schnelli
 
         base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,0);
         base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,5);

